src/noisepy/seis/application_modules/esyn_utils.py

```diff
@@ -35,13 +36,10 @@ def read_pyasdf(sfile, ccomp):
 
 ### -----
 # Function that Calculate Mean Square
-def msValue(arr, n):
-    square = 0
+def msValue(arr: np.ndarray, n: int) -> Tuple[np.ndarray]:
```

Comment:
A Tuple is not needed when there's a single value being returned, eg:
```suggestion
def msValue(arr: np.ndarray, n: int) -> np.ndarray:
```
Same for a few of the functions below.